### PR TITLE
nighthawk load testing

### PR DESCRIPTION
**Pull request by kumarabd**
Signed-off-by: kumarabd <abishekkumar92@gmail.com>

**Description**

Nighthawk load testing with latest binaries

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — CHANGES_REQUESTED** [review]
A few checks are in need of attention.

File: Dockerfile [hunk 1]
```diff
@@ -42,6 +42,14 @@ COPY --from=provider-ui /out /app/provider-ui/out
 COPY --from=wrk2 /wrk2 /app/cmd/wrk2
 COPY --from=wrk2 /wrk2/wrk /usr/local/bin
 COPY --from=nighthawk /nighthawk-go/apinighthawk/bin /usr/local/bin
+RUN curl -Lo nighthawk_service-ubuntu-amd64-v1.0.1.tar.gz https://github.com/layer5io/getnighthawk/releases/download/v1.0.1/nighthawk_service-ubuntu-amd64-v1.0.1.tar.gz \
```
Comment (leecalcote) [inline Dockerfile:45]: Pinning to a specific version seems like something that we don't want to do. Will you give this additional thought and circle back with an alternate strategy?

Comment (kumarabd) [reply Dockerfile:45]: sure

**Comment by leecalcote** [comment]
Go.sum conflict and lint check failures.

**Pull request merged**